app/main.ipynb

```diff
@@ -237,18 +311,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 23,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "sums_of_rows = np.sum(A, axis=1)\n",
+    "print(sums_of_rows) "
```

Comment:
The calculation of the sum of all elements in each row using `np.sum` with `axis=1` is correct and matches the task requirement.